model_analyzer/analyzer.py

```diff
@@ -98,11 +105,22 @@ def profile(self, client, gpus):
 
         self._profile_models()
 
+        if (self._state_manager.exiting()):
+            sys.exit(1)
```

Comment:
Is it okay to terminate without any message?

Reply:
Yes, the message is in the interrupt handler(). I've added a comment indicating this.